src/components/shared/forms/contextHelpLink.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Can you please test this component in Edge browser?

Reply:
What are you seeing? It looks acceptable to me.

Reply:
Just requested you to do so if you had missed to test it. We faced some challenges with Edge in my previous team for tooltips. 👍

Reply:
Gotcha. These are temporary tooltips... one of my features in 1811 will implement the real ones.  Though I'll likely start with this as the basis... so its definitely a good thing to check. Thanks for the reminder.